sympy/physics/mechanics/body.py

```diff
@@ -136,6 +136,10 @@ def __init__(self, name, masscenter=None, mass=None, frame=None,
         else:
             RigidBody.__init__(self, name, masscenter, frame, _mass, _inertia)
 
+    @property
+    def loads(self):
+        return self._loads
```

Comment:
I think that having `loads` as a mutable list is a good thing in this case. Users may want to remove loads from the list and manipulating the list is the easiest way to do so.

Reply:
In other words, a user should be able to add and remove loads manually and not need the new functions you've made. Those functions are really just convenience functions.

Reply:
I think we should keep it immutable else user may be able to append that list. But if it is immutable only way to append is by using functions, which would ensure that force is vector, point is a Point and all.

Reply:
Why is it bad for someone to append to that list? It isn't like something has to be calculated in Body when that list is modified. It is just a simple storage container.

Reply:
In joint, you'd have to recompute the joint's position and velocities if coordinate or speed was changed, thus we make it immutable to force the user to choose the coordinate and speed only on initialization. But the user may not know what all the loads are on initialization, then add them slowly in the problem construction. So `.load` has to be mutable. There is no reason to make the mutable `.load` a property unless you do something in it's setter method. That is, mutable attributes only need to be properties if there is a setter method. Body does nothing with `.load` so it should be a simply mutable attribute.

Reply:
But if we have `JointsMethod`, and then load changes on Body then it may affect result because loads are Body's attributes.